### PR TITLE
Remove examples with long runtimes

### LINKED ISSUE #430: Long example run times
Some of the examples have very long run times - especially the ones for `estimate_infections` which run a number of model variants that take about an hour on github actions. It might be good to keep this to a single example and move the variants to a vignette explaining/showing differences and possible advantages/disadvantages.

### PR DESCRIPTION
**Pull request by sbfnk**
Partly by storing some model results as data. See https://github.com/epiforecasts/EpiNow2/pull/458#issue-1915664824

Closes #430.

### CODE REVIEW

**Review by seabbs — COMMENTED** [review]
Nice. Do you think CRAN are going to be okay with this? In the past they have not been which is why `epinowcast` just `extdata` and loads in the example from a function.

**Comment by sbfnk** [comment]
Good question - also the increased size of `data/` might raise eyebrows. 

This also reminded me that we have example outputs in `inst` in `rbi` which didn't cause issues with CRAN. Will move it there.

**Review by seabbs — DISMISSED** [review]
LGTM

**Comment by sbfnk** [comment]
Source tarball is now 6.6 MB which should be fine for CRAN.

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # EpiNow2 1.4.9000
 
+## Package
+
+* Reduced the number of long-running examples.
```
Comment (seabbs) [inline NEWS.md:5]: Missing the standard PR, author and reviewer attribution we have used elsewhere.

**Pull request merged**